### ISSUE TITLE
Craft key messages for Privacy, Inclusion, Web literacy

### ISSUE THREAD

**Issue by OpenMatt** [MozillaFoundation/strategy#75]
Overall plan: https://docs.google.com/document/d/1fyNVt6gW-qt2VU6eovOBR0fkCisS2PmG-8g3VDIPROo/edit

We should ensure we have canonical language around what these issues are about and why they're important. 

Draft already started:
https://docs.google.com/document/d/1N4w5i3iPzzfzLgQitoBLDhjqfqdpLRhYlIXGAGh9jTk/edit
#### Roles

Project manager: @benrito
Consulted: @KevZawacki @HPaulJohnson 
Decider: @msurman

**Comment by OpenMatt** [comment]
This is an interesting example of how we could present these graphically: https://www.fordfoundation.org/work/challenging-inequality/our-approach/

**Comment by KevZawacki** [comment]
Ping @benrito, who I believe is working on the same / a parallel project.

**Comment by samanthaburton** [comment]
Yes, @benrito is working on issue briefs for all these - we may need to boil down to publicly-consumable key messages after those are complete, but let's just call it the same project for now.

**Comment by samanthaburton** [comment]
Update: 1st drafts created, [v1 summary paragraphs](https://docs.google.com/document/d/1Hf4Iaa4KdEx-rtqDUfAbO0POsM5K7Z3oNnoTjze576g/edit?usp=sharing) currently being circulated by me/Paul & Marketing for internal and external feedback.
